flink-cdc-common/src/main/java/org/apache/flink/cdc/common/schema/Selectors.java

```diff
@@ -29,8 +33,16 @@
 /** Selectors for filtering tables. */
 public class Selectors {
 
+    private static final Duration CACHE_EXPIRE_DURATION = Duration.ofHours(1);
+
     private List<Selector> selectors;
 
+    private final Cache<TableId, Boolean> cache =
```

Comment:
Maybe we can use `LoadingCache` here instead of checking / updating caches manually.

Reply:
IIUC both `Selector` and `Selectors` are meant to be effectively `final` and should not have any mutable states. Any comments @GOODBOY008?